cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't think this fixture is used. Can it be deleted?

Reply:
I'll delete for now but keep the directory as it'll be needed once we aren't hardcoding the title etc.

Reply:
We don't know when that might be. It maybe should be deleted, and just added in iteratively when it is needed.